README.md

```diff
@@ -1028,21 +1027,18 @@ class MyWidget extends WidgetBase<WidgetProperties> {
         // run your meta
         const html = this.meta(HtmlMeta).get('comment');
 
-        return v('div', { key: 'root', innerHTML: html });
+        return v('div', { key: 'comment', innerHTML: html });
     }
     // ...
 }
 ```
 
-Meta classes are provided with a few hooks into the widget, passed to the constructor:
+Extending the base class found in `meta/Base` will provide the following functionality:
 
-* `nodes` - A map of `key` strings to DOM elements. Only `v` nodes rendered with `key` properties are stored.
-* `requireNode` - A method that accepts a `key` string to inform the widget it needs a rendered DOM element corresponding to that key. If one is available, it will be returned immediately. If not, the widget will be re-rendered and if the node does not exist on the next render, an **error** will be thrown.
+* `has` - A method that accepts a `key` and returns a `boolean` to denote if that node exists in the rendered DOM.
```

Comment:
`if that` read a bit oddly => `if the node for the request key exist in the DOM`

Reply:
sure